app/src/main/java/app/revanced/integrations/requests/Requester.java

```diff
@@ -24,32 +25,52 @@ public static HttpURLConnection getConnectionFromRoute(String apiUrl, Route rout
         return connection;
     }
 
+    /**
+     * Parses the connection stream, and then and disconnects the {@link HttpURLConnection}
+     */
```

Comment:
Comments are a inconsistent in style, please make sure to use one style, preferably, the same as existing comments.

Reply:
I cleaned up the comments a bit